src/test/java/ru/lanwen/verbalregex/BasicFunctionalityUnitTest.java

```diff
@@ -563,7 +655,7 @@ public void zeroOreMoreSameAsAtLeast0() throws Exception {
         assertThat(regexWithOneOrMore, matchesTo(empty));
         assertThat(regexWithOneOrMore, matchesExactly(empty));
     }
-    
```

Comment:
Please avoid such changes

Reply:
Thanks for advise. Should I undo this modification

Reply:
I will be careful about that from next time. Thanks